cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -835,6 +835,14 @@
     </description>
   </property>
 
+  <property>
+    <name>log.kafka.topic.prefix</name>
+    <value>logs.user-v2</value>
+    <description>
+      Kafka log topic to publish logs
+    </description>
+  </property>
+
```

Comment:
Is this being used as a version string? ("v2") Perhaps add something to the property name or description so that it is known.